scripts/check_environment.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+i=0
+while [ $i -ne 10 ]
```

Comment:
IIUC we are going to loop 10 times waiting for the containers to be removed, sleeping 2 secs on each loop, right?

Reply:
@mdelapenya Yes, you understood correctly. I’ve noticed that containers were really removed from ‘docker ps’ output after N seconds( in average in ten seconds).

That’s why I decided to wait 20 seconds for the result. The script checks the Docker output 10 times every 2 seconds to speed up the process . And if the output is not empty after the loop, the script returns an error.